clients/ethereum/erc20ContractsHolder.go

```diff
@@ -0,0 +1,74 @@
+package ethereum
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-eth-bridge/clients/ethereum/contract"
+	"github.com/ElrondNetwork/elrond-eth-bridge/core"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	ethCommon "github.com/ethereum/go-ethereum/common"
+)
+
+type ArgsErc20SafeContractsHolder struct {
+	ethClient              bind.ContractBackend
```

Comment:
I think the argument's fields should always be exported, otherwise we can't use this in other packages.